egg/zoo/compo_vs_generalization/curriculum_games.py

```diff
@@ -38,8 +38,6 @@ def __getattr__(self, attrname):
             return getattr(self.game, attrname)
             
 
-
-
```

Comment:
Some weird and inconsistent line spacing here too. Again, consider autoformatting to handle this (if you plan to run autoformat do it via a separate PR to reduce clutter)